src/app/components/add-volunteers/add-volunteers.component.ts

```diff
@@ -52,80 +57,63 @@ export class AddVolunteersComponent implements OnInit {
       }),
       numberOfVolunteers: ['', Validators.min(1)],
       availableContent: [''],
-      days: this.fb.group({
-        1: [false],
-        2: [false],
-        3: [false],
-        4: [false],
-        5: [false],
-        6: [false],
-        7: [false],
-      }),
-      regions: this.fb.array(this.regions.map(region => this.fb.control(false))),
+      multiOccurrence: [''],
```

Comment:
Is `multiOccurrence` a boolean? If so, maybe the default value should be `false` instead of `''`.

Reply:
You are right